github3/gists/file.py

```diff
@@ -7,6 +7,8 @@
 """
 from __future__ import unicode_literals
 
+import requests
+
 from ..models import GitHubObject
```

Comment:
Below this you need to make `GistFile` inherit from `GitHubCore` not `GitHubObject`, which means chaning the import from `..models`.